concent_api/concent_api/settings/development.py

```diff
@@ -28,3 +28,7 @@
 STORAGE_CLUSTER_ADDRESS = 'http://devel.concent.golem.network/'
 
 GETH_CONTAINER_ADDRESS = 'http://localhost:8545'
+
+STORAGE_SERVER_INTERNAL_ADDRESS = 'http://devel.concent.golem.network/'
```

Comment:
Please don't use the address of the cluster. If won't work and if it would, it would not do us any good. This should probably be localhost or a container name or something. If anyone's going to have the storage server running locally it will be in a docker container.

How do you run the server locally? The development settings are meant to work out of the box with that.

Reply:
:+1: done

> How do you run the server locally? The development settings are meant to work out of the box with that.

By `manage.py runserver`.

Reply:
I am not running it yet, but I guess locally it should be on the same host as Concent ?

Reply:
So please do. @bartoszbetka has prepared a script that runs a local storage server just for that purpose. Really, please do not wait with testing this stuff until we get it all on the cluster. The tasks are defined such that you should be able to test each one separately with minimal setup (RabbitMQ container and maybe the storage server). If you can't, tell me, we'll find a solution. We don't want to rush all these tasks and then have a big headache integrating them at the end. You tested all the conductor tasks, locally too, right?

The server you run locally could be on `localhost` (if it uses `--net=host` when running the Docker container). A good development default would be whatever you need to put in this setting to connect to the sever started with that script. Definitely not the address of a real cluster :)